src/main/webapp/WEB-INF/web.xml

```diff
@@ -75,5 +75,10 @@
     <servlet-name>ActivityServlet</servlet-name>
     <url-pattern>/activity.jsp</url-pattern>
   </servlet-mapping>
+  
```

Comment:
I think you also need to add


   <servlet>
     <servlet-name>ProfileServlet</servlet-name>
     <servlet-class>codeu.controller.ProfileServlet</servlet-class>
   </servlet>